Code/Engine/RendererFoundation/Descriptors/Descriptors.h

```diff
@@ -139,14 +139,12 @@ struct EZ_RENDERERFOUNDATION_DLL ezGALVertexAttribute
 {
   ezGALVertexAttribute() = default;
 
-  ezGALVertexAttribute(ezGALVertexAttributeSemantic::Enum semantic, ezGALResourceFormat::Enum format, ezUInt16 uiOffset, ezUInt8 uiVertexBufferSlot,
-    bool bInstanceData);
+  ezGALVertexAttribute(ezGALVertexAttributeSemantic::Enum semantic, ezGALResourceFormat::Enum format, ezUInt16 uiOffset, ezUInt8 uiVertexBufferSlot);
 
   ezGALVertexAttributeSemantic::Enum m_eSemantic = ezGALVertexAttributeSemantic::Position;
   ezGALResourceFormat::Enum m_eFormat = ezGALResourceFormat::XYZFloat;
   ezUInt16 m_uiOffset = 0;
   ezUInt8 m_uiVertexBufferSlot = 0;
-  bool m_bInstanceData = false;
```

Comment:
Why has this been removed? We don't use it at the moment but will in the near future.

Reply:
It's not supported on all platforms, and since we don't use it at the moment, I find it unnecessary to expose it.
I did keep the relevant D3D and Vulkan implementation code in (commented out), in case we do want to re-add it, but as long as we don't use it, I would remove it from the interface.

Reply:
I would be very surprised if there is a platform that doesn't support it. It has been around since Dx9 and very old opengl version. And yes webgpu does support it.

Reply:
We store it on a per-vertex stream basis (see code above). In WebGPU this is a single value on an entire VertexBufferLayout, which combines all streams.
So yes, it supports it, but no, not the way that we have it in the API.